utils/constants.js

```diff
@@ -0,0 +1,7 @@
+const CONSTANTS = {
+  TimestampExpiry: 300,
+  baseUrl: "https://yayawallet.com/api/en",
```

Comment:
have a means to override this, or use env variable. This way the developers can switch between environments  without changing the SDK